src/Tribe/Attendees_Table.php

```diff
@@ -65,7 +65,7 @@ public function __construct( $args = array() ) {
 	public function get_table_columns() {
```

Comment:
Leaving this note for you @cliffordp to double check that the `product_id` and `ID` values are correct when you merge the conflicts over.